packages/tailwindcss/src/utils/decode-arbitrary-value.ts

```diff
@@ -1,13 +1,16 @@
+import * as ValueParser from '../value-parser'
 import { addWhitespaceAroundMathOperators } from './math-operators'
 
 export function decodeArbitraryValue(input: string): string {
-  // We do not want to normalize anything inside of a url() because if we
-  // replace `_` with ` `, then it will very likely break the url.
-  if (input.startsWith('url(')) {
-    return input
+  // There's definitely no functions in the input, so bail early
+  if (input.indexOf('(') === -1) {
+    return convertUnderscoresToWhitespace(input)
   }
```

Comment:
This check was moved up from being inside the `addWhitespaceAroundMathOperators`, since neither the `url()` nor the `var()` handling need to run if the arbitrary value has no parenthesis.